wwwapp/management/commands/populate_with_test_data.py

```diff
@@ -66,8 +66,8 @@ def fake_user(self) -> (User, UserProfile):
     """
     Creates and returns a fake random article with a random edit history
     """
-    def fake_article(self, users) -> Article:
-        article = Article(name=self.fake.uri_page(),
+    def fake_article(self, users, sequence) -> Article:
```

Comment:
The sequence argument should be optional (don't append a number if not specified)